### PR TITLE
[5.4] Make RequestGuard Macroable

### PR DESCRIPTION
**Pull request by hezhiying**
for RequestGuard Class use Illuminate\Support\Traits\Macroable;
Similar to SessionGuard Class,Allows RequestGuard Class expansion Method.

### CODE REVIEW

**Comment by lucasmichot** [comment]
Tests ?

File: src/Illuminate/Auth/RequestGuard.php [hunk 1]
```diff
@@ -4,10 +4,11 @@
 
 use Illuminate\Http\Request;
 use Illuminate\Contracts\Auth\Guard;
+use Illuminate\Support\Traits\Macroable;
 
 class RequestGuard implements Guard
 {
-    use GuardHelpers;
+    use GuardHelpers,Macroable;
```
Comment (ntzm) [inline src/Illuminate/Auth/RequestGuard.php]: Needs a space after the comma

**Comment by ntzm** [comment]
Could do with a better title for the PR!

File: src/Illuminate/Auth/RequestGuard.php (hunk 1 above)
Comment (BrandonShar) [inline src/Illuminate/Auth/RequestGuard.php]: psr-2 says `There MUST be one use keyword per declaration`

Comment (lucasmichot) [inline src/Illuminate/Auth/RequestGuard.php]: True @BrandonShar but in this case this is not a declaration.

Comment (BrandonShar) [inline src/Illuminate/Auth/RequestGuard.php]: doh, read too fast!

**Comment by hezhiying** [comment]
Thank you, my English is not good, sorry. I am going to optimize the code tomorrow, I am now 1 o'clock now. I was doing api internal call need to expand some of the methods and found that not like SessionGuard, I Pull Request reasonable?

**Pull request merged**